### PR TITLE
feat: Introduced PR Creator and EkLine triggerer

### PR DESCRIPTION
**Pull request by deepakjoshi-ekline**
Ticket : [Link](https://ekline.atlassian.net/jira/software/projects/EK/boards/1?assignee=712020%3Abf39a77b-882a-43e0-98b9-696d31875731&selectedIssue=EK-555)

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -60,6 +74,7 @@ elif [ "$CI" = "true" ] && [ -n "$BITBUCKET_BUILD_NUMBER" ]; then
   head_branch="${BITBUCKET_BRANCH}"
   export INPUT_REPORTER='bitbucket-code-report'
   export INPUT_FILTER_MODE='nofilter'
+  triggering_actor="$BITBUCKET_STEP_TRIGGERER_UUID"
```
Comment (arun-ekline) [inline entrypoint.sh:77]: How much work is it to get the PR creator for Bitbucket?

**Review by arun-ekline — APPROVED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (deepakjoshi-ekline) [reply entrypoint.sh:77]: 2 story points .

Comment (arun-ekline) [reply entrypoint.sh:77]: Tracked at https://ekline.atlassian.net/browse/EK-607

**Comment by arun-ekline** [comment]
We still need to update the BigQuery table and the backend service, is that right?

**Comment by deepakjoshi-ekline** [comment]
> We still need to update the BigQuery table and the backend service, is that right?

We have already done that to persist this change in BigQuery

**Review by arun-ekline — APPROVED** [review]

**Pull request merged**